script/migrate-to-device-urn.bash

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# The purpose of this script is to backpopulate the device_urn column
+# as documented in issue 540. This is one step in migrating from
+# device_id to device_urn as the primary device identifier.
+
+errmsg() {
+    IFS=' '
+    printf '%s\n' "$*" 1>&2
+}
+
+psql_cmd='psql -U safecast ingest-solarcast_development -c'
```

Comment:
I'd make this a function. Pretty sure this usage of `$psql_cmd` won't pass shellcheck.

Reply:
Good idea to run shellcheck, it found a couple of errors in reset-db.bash. Apparently it didn't mind this, but I will make it a function, it needs to be more flexible anyway when I run against other environments.